storage/db-tool/src/replay_verify.rs

```diff
@@ -92,6 +92,7 @@ impl Opt {
                     process::exit(2);
                 },
                 _ => {
+                    info!("ReplayVerify coordinator exiting with error: {:?}", e);
```

Comment:
https://github.com/aptos-labs/aptos-core/blob/2905166666992084c37498a6e67a590bcfa300f0/storage/backup/backup-cli/src/coordinators/replay_verify.rs#L84

The coordinator prints this already?

Reply:
this print is just to confirm we reach this place